aten/src/ATen/core/DeprecatedTypeProperties.h

```diff
@@ -42,6 +42,10 @@ class TORCH_API DeprecatedTypeProperties {
     return backendToDeviceType(backend_) == kCUDA;
   }
 
+  bool is_mlc() const {
+    return backendToDeviceType(backend_) == kMLC;
+  }
+
```

Comment:
As the name of this class suggests, this is deprecated, no need to add MLC here.